application/backend/src/main/java/io/typerefinery/websight/models/components/BaseComponent.java

```diff
@@ -137,6 +130,23 @@ public String getComponentClassNames() {
         return "";
     }
 
+    private Map<String, String> gridConfig = new HashMap<String, String>() {
+        {
+            put("lgColSize", "col-lg-");
+            put("mdColSize", "col-md-");
+            put("smColSize", "col-sm-");
+        }
+    };
+
+    private Map<String, String> textAlignmentConfig = new HashMap<String, String>() {
```

Comment:
can this be in style class?

Reply:
I have moved the all config things to StyledGrid Classes.